src/internal/fiberRuntime.ts

```diff
@@ -744,6 +744,9 @@ export class FiberRuntime<in out E, in out A> implements Fiber.RuntimeFiber<E, A
   reportExitValue(exit: Exit.Exit<E, A>) {
     if (_runtimeFlags.runtimeMetrics(this._runtimeFlags)) {
       const tags = this.getFiberRef(core.currentMetricLabels)
+      const startTimeMillis = this.id().startTimeMillis
+      const endTimeMillis = new Date().getTime()
```

Comment:
**Question** 
In a previous conversation somewhere, I think you had suggested using `Date.now`?

Reply:
That is in the other PR :)